MapboxDirections/MBVisualInstructionComponent.swift

```diff
@@ -52,16 +60,17 @@ open class VisualInstructionComponent: NSObject, NSSecureCoding {
             imageURL = URL(string: "\(baseURL)@\(Int(scale))x.png")
         }
         
-        self.init(text: text, imageURL: imageURL)
+        self.init(text: text, imageURL: imageURL, isDelimiter: isDelimiter)
     }
     
     /**
      :nodoc:
      Initialize A `VisualInstructionComponent`.
      */
-    @objc public init(text: String?, imageURL: URL?) {
+    @objc public init(text: String?, imageURL: URL?, isDelimiter: Bool) {
```

Comment:
To avoid argument proliferation, let’s leave this argument out and set the `isDelimiter` property to `false` by default. It’s variable, so the caller can set it to `true` right after creating the object.